src/documents/models.py

```diff
@@ -266,6 +281,13 @@ def __str__(self):
             return "{}: {}".format(created, self.correspondent or self.title)
         return str(created)
 
+    def save(self, *args, **kwargs):
```

Comment:
Thank you for doing this in `.save()` and not in a signal.  This is much more appropriate.